components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -241,6 +246,7 @@ struct ActionGroup {
 struct Conversation {
   string uuid;
   string title;
+  // Set by the LLM after detecting the user's language in the first message
```

Comment:
leftover comment

Reply:
Fixed in a63bdca4187730e71a9d70be8a33135c1951795d